pythonforandroid/recipes/freetype/__init__.py

```diff
@@ -1,44 +1,118 @@
 from pythonforandroid.toolchain import Recipe
+from pythonforandroid.logger import shprint, info
 from pythonforandroid.util import current_directory
-from pythonforandroid.logger import shprint
-from os.path import exists, join, realpath
+from os.path import exists, join
+from multiprocessing import cpu_count
 import sh
 
 
 class FreetypeRecipe(Recipe):
+    """The freetype library it's special, because has cyclic dependencies with
+    harfbuzz library, so freetype can be build with harfbuzz support, and
+    harfbuzz can be build with freetype support. This complicates the build of
+    both recipes because in order to get the full set we need to compile those
+    recipes several times:
+        - build freetype without harfbuzz
```

Comment:
Do we do this part for the full set? It feels to me this one is only when we have `freetype` only in the requirements, right?
Thinking because of that line of code later `if not harfbuzz_in_recipes:`